aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -151,5 +151,27 @@ Tensor quantized_clone(const Tensor& self) {
   return dst;
 }
 
+bool quantized_equal(const Tensor& self, const Tensor& other) {
```

Comment:
Do we define equality as the tensors being absolutely equal, or numerically equal? If the latter is true, we need to check if dequantized values are equal, no matter the scale/zero_point.

Reply:
@zafartahirov we need both. This quick implementation is used for situations where we are checking tensor identity for memoization, which is currently implemented by calling equal(). It is an error to mark tensors that are numerically equivalent in the reals but have differing quantization parameters in this scenario.